src/checkers/htmlchecker.py

```diff
@@ -109,11 +109,10 @@ def check(self, external_data):
 
         self._update_version(external_data, latest_version, abs_url)
 
-    def _update_version(
-        self, external_data, latest_version, latest_url, follow_redirects=True
-    ):
+    def _update_version(self, external_data, latest_version, latest_url):
```

Comment:
This method is used by inherited checkers. Instead of removing the arg, change it's default value (all inherited checkers set it to `False` explicitly anyway).